.gitignore

```diff
@@ -38,7 +39,6 @@
 !/.travis.yml
 !/.yardopts
 !/CODEOFCONDUCT.md
-!/CONTRIBUTING.md
 !/LICENSE.txt
 !/README.md
 !/SUPPORTERS.md
```

Comment:
Let's also consider moving this file out of the root and into the `share/doc/homebrew` while we're moving stuff around.